### PR TITLE
Add test for digits only room lock

### PR DESCRIPTION
**Pull request by saghul**
Depends on https://github.com/jitsi/jitsi-meet/pull/7372

### CODE REVIEW

File: src/test/java/org/jitsi/meet/test/LockRoomDigitsOnlyTest.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+     */
+    @Test
+    public void lockRoomWithDigitsOnly()
+    {
+        ensureOneParticipant(
+            getJitsiMeetUrl().appendConfig(
+                "config.roomPasswordNumberOfDigits=5"));
+
+        // just in case wait
+        TestUtils.waitMillis(1000);
```
Comment (damencho) [inline src/test/java/org/jitsi/meet/test/LockRoomDigitsOnlyTest.java:46]: Is this needed? 
... You can do getParticipant1().waitToJoinMUC();

File: src/test/java/org/jitsi/meet/test/LockRoomDigitsOnlyTest.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+        // Set a valid numeric password.
+        securityDialog.addPassword("12345");
+        TestUtils.waitMillis(1000);
+        securityDialog.close();
+    
+        TestUtils.waitMillis(1000);
+        securityDialog.open();
+
+        assertTrue(securityDialog.isLocked());
```
Comment (damencho) [inline src/test/java/org/jitsi/meet/test/LockRoomDigitsOnlyTest.java:79]: I would love to get rid of all the waits ... We are trying to clear those, as during the years we have a lot of those (I personally am responsible for a lot of those) some without explanation ... and it is hard to understand after some time ...

**Comment by damencho** [comment]
Other than the waits LGTM

**Comment by damencho** [comment]
And the failure 😛

File: src/test/java/org/jitsi/meet/test/LockRoomDigitsOnlyTest.java (hunk 2 above)
Comment (saghul) [reply src/test/java/org/jitsi/meet/test/LockRoomDigitsOnlyTest.java:79]: Sounds good, will do!

**Comment by saghul** [comment]
> And the failure 😛

Hehe, the problem is that the tests will fail unless the config override works. So, do you have a suggestion on how to handle that?

**Comment by damencho** [comment]
I'm merging with a plan to remove those waits as part of my next PR which adds some modifications to security dialog.

**Pull request merged**